app/models/user.js

```diff
@@ -26,6 +26,14 @@ exports.getModel = db => {
       password: {
         type: Sequelize.STRING,
         allowNull: false
+      },
+      auth_code_validation: {
```

Comment:
I know im being a bit repetitive hahaha but, don't forget to change this to camelcase (and is_administrator too)